### ISSUE TITLE
NVDA 18.3 doesn't jump by headings 'h' in Crome

### ISSUE THREAD

**Issue by mythral0** [nvaccess/nvda#8761]
### Steps to reproduce:
1. Install NVDA 18.3
2. Once installed and started, open google crome
3. go to www.google.com
4. press 'e' to find the edit box
5. enter a search term
6. try to navigate results by heading
7. it doesn't work.

### Actual behavior:
navigating on a webpage in crome using headings 'h' doesn't work.

### Expected behavior:
I expect on press of 'h' for NVDA to junp to the next heading and read it.

### System configuration:
win10

#### NVDA Installed/portable/running from source:
NVDA installed

#### NVDA version:
18.3.

#### Does the issue still occur after restarting your PC?
yes

#### Have you tried any other versions of NVDA?
18.21 works fine.

**Comment by PratikP1** [comment]
Can you provide the version of Chrome you’re using? I am unable to replicate this behavior with canary Version 71.0.3555.0 and the latest NVDA alpha.

**Comment by mythral0** [comment]
Crome Version 69.0.3497.100 (Official Build) (64-bit)

**Comment by leonardder** [comment]
Have you restarted your system since this occured? Also, what happens if you press h while in input help (NVDA+1)? Does it appropriately report "Moves to the next heading"?

Also, is there anything error like in NVDA's log?

**Comment by Adriani90** [comment]
@mythral0 are you still having this issue with NVDA 2018.4.1 and last version of Chrome?

**Comment by Adriani90** [comment]
We did not have any updates from the initial author and I am not able to reproduce this issue on Chrome 79 with NVDA 2019.3 beta 1. I did also not hear any further user reports confirming this so far. Closing as works for me.
If you have this issue still, please comment on it and we can reopen it. Thanks.

**Comment by avprod1** [comment]
Sorry if I am reporting this in the wrong report. 
Chrome Version 87.0.4280.88 (Official Build) (64-bit)
NVDA 2017.1 (sorry, corporate locks down any updates). Sitting on a link. single character commands h and g only report the character. <capslock>+<space> has no impact. <capslock>+1, then h reports the character h. <shift>+h reports CapH. This is intermittant.
